ui/src/main/webapp/src/app/project/migration-project.service.ts

```diff
@@ -42,7 +46,8 @@ export class MigrationProjectService extends AbstractService {
 
         if (event.isTypeOf(NewExecutionStartedEvent)) {
             monitoredProject.executions.push((<NewExecutionStartedEvent>event).execution);
-        } else if (event.isTypeOf(ExecutionUpdatedEvent)) {
+        }
+        else if (event.isTypeOf(ExecutionUpdatedEvent)) {
```

Comment:
Could we please switch to the other convention? The current "convention", designed by comitee, makes the code unreadable for me. What is more important - whether someone reading the code can read it conveniently or following a convention created by people unrelated to us and the project?